src/cbmc/cbmc_solvers.cpp

```diff
@@ -189,8 +187,6 @@ class cbmc_solver_with_filet:public cbmc_solverst::solvert
   ~cbmc_solver_with_filet()
   {
     // delete the prop before the file
-    delete prop_conv_ptr;
```

Comment:
May I dare to say that this is a bit of a broken design? Multiple pointers that need to be allocated and de-allocated in a certain sequence, with no well-defined ownership. Shouldn't object A referencing object B assume ownership of B, and therefore do the cleanup? Yes, this will end up as a bit more work than @mgudemann may have set out to do on this, but we should rather be re-architecting than running around with memory leaks.